recipes/ctat-mutations/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ctat-mutations" %}
+{% set version = "1.0" %}
+{% set build = "1" %}
```

Comment:
Thanks for contributing a recipe for `ctat-mutations`. The build number for a new package [can start at zero](https://bioconda.github.io/guidelines.html#buildnum).

Reply:
I contributed another recipe ctat-lncrna a few days back and we used the same build number, to keep all ctat tools in sync I would like this recipe to start at build number at 1. Hope this is ok?